web/packages/shared/components/MenuLogin/MenuLogin.tsx

```diff
@@ -75,12 +75,14 @@ export const MenuLogin = React.forwardRef<MenuLoginHandle, MenuLoginProps>(
     return (
       <React.Fragment>
         <ButtonBorder
+          width={props.width}
```

Comment:
This makes the connect button in the database table too wide.

I'd inspect if this affects other places which use `MenuLogin` as well, an if so, add a prop such as `alignButtonWidthToMenu` or something like this? If only the unified resources view needs this, then I'd make it optional and false by default.

![database table in Connect](https://github.com/gravitational/teleport/assets/27113/01a2efb0-b5ac-4130-9b4f-b7496a85f09e)

Reply:
Good call thanks. I was checking Connect as writing this but the width came last and I must have not checked it after that. 

I added the `alignButtonWidthToMenu` because I think it makes sense. Then set to false by default and only updated the unified menu buttons with the prop 👍